crates/uv-resolver/src/pubgrub/dependencies.rs

```diff
@@ -11,21 +11,23 @@ use crate::constraints::Constraints;
 use crate::overrides::Overrides;
 use crate::pubgrub::specifier::PubGrubSpecifier;
 use crate::pubgrub::PubGrubPackage;
-use crate::resolver::Urls;
+use crate::resolver::{Locals, Urls};
 use crate::ResolveError;
 
 #[derive(Debug)]
 pub struct PubGrubDependencies(Vec<(PubGrubPackage, Range<Version>)>);
 
 impl PubGrubDependencies {
     /// Generate a set of `PubGrub` dependencies from a set of requirements.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Why do we have this lint on? I feel like we just ignore it.

Reply:
We should probably have fewer arguments 😂